packaging/datadog-agent/source/install_agent.sh

```diff
@@ -81,6 +81,8 @@ elif [ -f /etc/system-release -o "$DISTRIBUTION" == "Amazon" ]; then
 # Arista is based off of Fedora14/18 but do not have /etc/redhat-release
 elif [ -f /etc/Eos-release -o "$DISTRIBUTION" == "Arista" ]; then
     OS="RedHat"
+elif [ -f /etc/issue -o "$DISTRIBUTION" == "SUSE" ]; then
```

Comment:
`/etc/issue` seems to be present on a lot of platforms (it's present on both debians and rhels at least from what I've checked).

It looks like `/etc/SuSE-release` would be better to detect the distribution, what do you think?